lib/recog/fingerprint.rb

```diff
@@ -136,6 +136,13 @@ def match(match_string)
       end
     end
 
+    # After performing interpolation, remove temporary keys from results
+    result.each_pair do |k, _|
+      if k.start_with?("_tmp.")
```

Comment:
Use single quoted strings since they are more common in this module.

Reply:
Fixed.